test/jdk/java/awt/dnd/NoFormatsDropTest/NoFormatsDropTest.java

```diff
@@ -94,17 +94,14 @@ public void init() {
 
         frame.getToolkit().addAWTEventListener(this, AWTEvent.MOUSE_EVENT_MASK);
         frame.pack();
-        frame.setLocationRelativeTo(null);
         frame.setVisible(true);
-        frame.validate();
     }
 
     public void start() throws InterruptedException, AWTException,
             InvocationTargetException {
         try {
             Robot robot = new Robot();
-            robot.setAutoWaitForIdle(true);
-            robot.delay(FRAME_ACTIVATION_TIMEOUT);
+            Thread.sleep(FRAME_ACTIVATION_TIMEOUT);
```

Comment:
Curious to know the difference of using `Thread.sleep()` instead of `robot.delay()` here ?

Reply:
Initially, I thought ```Thread.sleep``` may have been the cause since there are some synchronization lines. But, I think robot.delay does the same behavior here. So, to keep the modern test format, I reverted this to robot.delay